source/EvoPlanet/EvoPlanet.Server/Program.cs

```diff
@@ -27,7 +27,7 @@ public static void Main(string[] args)
             builder.Services.AddSingleton<PlanetService>();
             // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
             builder.Services.AddEndpointsApiExplorer();
-            builder.Services.AddSwaggerGen();
+            //builder.Services.AddSwaggerGen();
```

Comment:
It is best to minimize comments in code (Clean Code principles). Comments usually don't get updated as diligently as code, consequently it results in more confusion than clarity.
If you keep comments while developing on your local branch, because you might still them, fine. However, they should not make it to the Main/master branch, and should not be present in a pull request.

If you need this code later, you will find it in the history of the source control. Remove this comment.